packages/frontend/src/view/ApiDefs/Form/CacheTransformsInvalidateOperations.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useFieldArray } from 'react-hook-form';
+
+import { CacheTransformsInvalidateOperations as Props } from '../../../types';
+import { ObjectArray } from '../../Form';
+
+export const CacheTransformsInvalidateOperations: React.FC<Props> = ({
+  nestIndex,
+  control,
+  errors,
+  register,
+}) => {
+  const name = `cache[${nestIndex}].invalidate.effectingOperations`;
+  const { fields, remove, append } = useFieldArray({
+    control,
+    name,
+  });
+
+  return (
+    <ObjectArray
+      title="Invalidate Operations"
```

Comment:
Invalidation rules

Reply:
Can we display helperTexts for operation and matchKey here?

They would be:
- operation: helperText="Operation name (i.e. Mutation.createUser) which will trigger the invalidation"
- matchKey: helperText="Must match the Cache key which should be invalidated."

If possible, also add the same helper "?" icon with the list of available cache key variables.